apps/providers/test/unit/providers/date_test.exs

```diff
@@ -32,7 +33,9 @@ defmodule Providers.DateTest do
 
       result = Providers.Date.provide("1", %{format: format, offset_in_days: offset_in_days})
 
-      assert Timex.now() |> Timex.add(Timex.Duration.from_days(offset_in_days)) |> Timex.format!(format) == result
+      assert Timex.now()
```

Comment:
what if this runs right at midnight? :D

Reply:
Dunno. I suspect its not worth trying to solve that particular edge case